### PR TITLE
Show sentiment analysis results

### LINKED ISSUE #27: Feed does not show rich text. It shows html tags instead.

### PR DESCRIPTION
**Pull request by nbaldy**
Show the Sentiment Analysis score in the header of the message containers in the user page and feed
Also fixes #27 (showing html in public feed rather than rich text)

### CODE REVIEW

**Comment by drewbern** [comment]
I'm not able to verify the Sentiment Analysis score due to an error caused when I try to post a message. I looked in the Sentiment Analysis group and saw your suggestion of doing a mvn clean, and tried it but to no avail. Any suggestions on getting it to work?

**Comment by drewbern** [comment]
@nbaldy I see the sentiment scores! The score for any message I post is [0], is that the correct behavior?

**Comment by nbaldy** [comment]
> @nbaldy I see the sentiment scores! The score for any message I post is [0], is that the correct behavior?

What are you saying in your messages? They shouldn't all be 0's. In @acolwell 's latest PR, the "default" sentiment score if an error is thrown is 0... but that shouldn't affect this PR, as I haven't merged it yet... So I'm honestly not sure what's going on... 

Maybe if you have some time I can walk you through the google account/authentication procedure, because I think that is the error, that this is not set up for your account. Honestly, what might be best is pushing to the "real" appengine as its authentication is set up... but then the changes go live, and so if my code doesn't work, the "live" version is broken for a while

**Comment by drewbern** [comment]
Yeah sure, I'd appreciate help with the authentication! And I wouldn't mind trying it on the real appengine and then just updating it with master after to restore it. I think you might have to merge Aaron's fix today in master into your branch though so it will work (on my end at least). Locally, I could only see the messages by first posting using the master branch with the fix, then switching to this branch and looking at those messages.

**Comment by nbaldy** [comment]
> Yeah sure, I'd appreciate help with the authentication! And I wouldn't mind trying it on the real appengine and then just updating it with master after to restore it. I think you might have to merge Aaron's fix today in master into your branch though so it will work (on my end at least). Locally, I could only see the messages by first posting using the master branch with the fix, then switching to this branch and looking at those messages.

Sounds good! I will do that merge, is now a good time for you to walk through the authentication stuff real fast? If not I will probably be available tomorrow night

**Comment by drewbern** [comment]
Yeah, now is fine, send me a message on chat whenever you're ready!

**Comment by drewbern** [comment]
Let me know when you fix the merge break, and I'll go ahead and approve it the PR!

**Comment by nbaldy** [comment]
I think this is fixed!

**Review by drewbern — APPROVED** [review]
LGTM

File: src/main/webapp/WEB-INF/user.jsp [hunk 1]
```diff
@@ -51,7 +51,9 @@
     %>
           <div class="message-div">
             <div class="message-header">
-              <%= messages.get(i).getUser() %> - <%= new Date(messages.get(i).getTimestamp()) %>
+              User: <%= messages.get(i).getUser() %> -
+              Time: <%= new Date(messages.get(i).getTimestamp()) %> -
+              Sentiment Score: <%= messages.get(i).getSentimentScore() %>
```
Comment (drewbern) [inline src/main/webapp/WEB-INF/user.jsp:56]: I like the score descriptor, not sure if you wanted to add it to the feed messages as well or not, but if not, LGTM!

**Review by drewbern — APPROVED** [review]

**Comment by drewbern** [comment]
#36

File: src/main/webapp/WEB-INF/user.jsp (hunk 1 above)
Comment (nbaldy) [reply src/main/webapp/WEB-INF/user.jsp:56]: I'll go ahead and add that to the feed, too. Thanks!

Comment (nbaldy) [reply src/main/webapp/WEB-INF/user.jsp:56]: Actually, that's still using HTML so I might need to add some extra logic to the servlet. So, I think I will merge without it, and update that in a separate PR after we discuss if we want the feed to be SSR or not

**Pull request merged**